### DISCUSSION TITLE
Add journeys

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [collab-community/journey-book#34] [General]
@Adarsh-jaiss @vikasganiga05 Please create your journey in the [journeys directory](https://github.com/Adarsh-jaiss/journey-book/tree/main/journeys) as a .md file asap.

**Comment by Adarsh-jaiss** [comment]
-- okay, I am goona make one 😃

**Reply by KendallDoesCoding** [reply]
Cool! Please also setup GitHub Pages as per discussed in #32

**Reply by Adarsh-jaiss** [reply]
Actually I don't have any idea about that, but let me see, if I can do something

-- sorry for the late replies, since my jee exams are on my head after in a week, so I was busy preparing for my exams :)

**Reply by KendallDoesCoding** [reply]
> Actually I don't have any idea about that, but let me see, if I can do something

Ok cool.

> -- sorry for the late replies, since my jee exams are on my head after in a week, so I was busy preparing for my exams :)

No worries, JEE is very hard, all the best!

**Reply by Adarsh-jaiss** [reply]
Yeah thanks for understanding :)
